Vic2ToHoI4/Source/HOI4World/HoI4Country.h

```diff
@@ -219,6 +219,8 @@ class Country
 	const auto& getGuaranteed() const { return guaranteed; }
 	void addGuaranteed(std::string guaranteedTag) { guaranteed.push_back(guaranteedTag); }
 
+	bool isEligibleEnemy(std::string target);
```

Comment:
Put [[nodiscard]] in front of bool. Mark it const as you're not altering anything.